xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -1339,10 +1601,69 @@ This check is conceptually similar to `(void*)(&left) < (void*)(&right)`.
         <Parameter Name="value" Type="T" RefType="out" Index="0" FrameworkAlternate="net-5.0;net-6.0;net-7.0" />
       </Parameters>
       <Docs>
-        <typeparam name="T">The type of the uninitialized object.</typeparam>
-        <param name="value">The uninitialized object.</param>
-        <summary>Bypasses definite assignment rules for a given value.</summary>
-        <remarks>To be added.</remarks>
+        <typeparam name="T">The type of the reference.</typeparam>
+        <param name="value">The reference whose initialization should be skipped.</param>
+        <summary>Bypasses definite assignment rules for a given reference.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This is typically used to avoid double-assignment when initializing union-type structs. Consider the following example, which produces a C# compiler error.
```

Comment:
```suggestion
This method is typically used to avoid double-assignment when initializing union-type structs. Consider the following example, which produces a C# compiler error.
```